cmills22/BugBear
# BB-102 Adding javascript to information form

**What ticket is this for?**

BB-102

**What do we need to know to understand these changes?**

The javascript added is to make the information form "pop up" when the "Add Task" button is clicked, and empty all of the text fields every time the form pops up. Also when the "Cancel" button on the form is clicked the information form will become hidden.

**What does this PR do?**

Give "popup" functionality to the information form.

**How can we manually test your changes?**

1. Click the "Add Task" button, the form should pop up with all text fields empty.
2. Click the "Cancel" button on the form, and the form should disappear.

**Are there any blockers to merging this ticket?**

This ticket is dependent on BB-100 and must be merged after. Also if any changes have been made to BB-100, the changes must also be applied to this ticket

**What are you not sure of?**

If this ticket will require any changes before merging as it builds off of BB-100

**Relevant Documentation:**

Prototype Use Cases
https://eventmobi.atlassian.net/wiki/spaces/ENG/pages/415629357/BB+-+Use+Cases
